coq/coq-syntax.el

```diff
@@ -1297,6 +1297,9 @@ It is used:
   "*Font-lock table for Coq terms.")
 
 
+(defconst coq-show-proof-diffs-regexp
+  "\\bShow Proof\\(?: Diffs\\| Diffs removed\\)?\\.\\b")
```

Comment:
the [`\\b`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Regexp-Backslash.html#Regexp-Backslash) token seems a bit off-topic here:

```suggestion
  "\\`Show Proof\\(?: Diffs\\| Diffs removed\\)?\\.\\'")
```

Reply:
This suggested regex doesn't work for me, it doesn't match `Show Proof.` for example. I don't know why.

Reply:
@CyrilAnac can you give more details? what do you mean by "doesn't work"? On my setup (emacs 26.1, GNU/Linux), this regex indeed returns a non-nil value:

```
(string-match-p "\\`Show Proof\\(?: Diffs\\| Diffs removed\\)?\\.\\'" "Show Proof.")
```

Reply:
Oh, right. I didn't understand the point of these anchors. With this test the result is 0 like expected.